esp/src/src-react/components/controls/Grid.tsx

```diff
@@ -179,6 +246,42 @@ export const FluentStoreGrid: React.FunctionComponent<FluentStoreGridProps> = ({
     />;
 };
 
+interface FluentGridProps {
+    data: any[],
+    primaryID: string,
+    alphaNumColumns?: { [id: string]: boolean },
```

Comment:
I've never looked at this I guess. But what's the significance of the boolean? Like what would { Name: false, ... } mean? Just thinking it could be string[] instead and the presence of the column id would suffice to imply it should be sorted alphanumerically.

Reply:
Name: false would be the same as ommitting Name.  I suspect I went with the object format over an array as the lookup would be slightly quicker (assuming it was a longer list!)